doc/auto_techsupport_and_coredump_mgmt.md

```diff
@@ -261,26 +333,26 @@ config auto-techsupport-feature delete restapi
 
 ```
 admin@sonic:~$ show auto-techsupport global
-STATE      RATE LIMIT INTERVAL (sec)    MAX TECHSUPPORT LIMIT (%)    MAX CORE SIZE (%)       SINCE
--------  ---------------------------   --------------------------    ------------------  ----------
-enabled                          180                        10.0                   5.0   2 days ago
+STATE      RATE LIMIT INTERVAL (sec)    MAX TECHSUPPORT LIMIT (%)    MAX CORE SIZE (%)   MEM THRESHOLD (%)       SINCE
+-------  ---------------------------   --------------------------    ------------------  ------------------  ----------
+enabled                          180                        10.0                   5.0                 90.0   2 days ago
 
 admin@sonic:~$ show auto-techsupport-feature 
-FEATURE NAME    STATE       RATE LIMIT INTERVAL (sec)
---------------  --------  --------------------------
-bgp             enabled                          600
-database        enabled                          600
-dhcp_relay      enabled                          600
-lldp            enabled                          600
-macsec          enabled                          600
-mgmt-framework  enabled                          600
-nat             enabled                          600
-pmon            enabled                          600
-radv            enabled                          600
-restapi         disabled                         800
-sflow           enabled                          600
-snmp            enabled                          600
-swss            disabled                         800
+FEATURE NAME    STATE     MEM THRESHOLD (%)     RATE LIMIT INTERVAL (sec)
+--------------  --------  ------------------  --------------------------
+bgp             enabled                 90.0                         600

```

Comment:
@stepanblyschak  are there any default values set when you enable the global param? if so, please state